src/Core/DynamicMetadata/SubjectArea.php

```diff
@@ -32,7 +32,7 @@ class SubjectArea implements DynamicMetadataInterface
      * @param int $x      X-point
      * @param int $y      Y-point
      */
-    public function __construct($x, $y, $width, $height)
+    public function __construct($x, $y, $width = 1, $height = 1)
```

Comment:
this should also be explained in the phpdoc. the order of the parameters in the phpdoc is wrong, btw.

while there is some redundancy, imho the phpdoc should explain a minimum of how to use the rokka api. no fancy explanations, but just saying that the subject area can be a dot or a rectangular area.

btw, is $width | $height < 1 allowed? if not we could validate this in this method and throw an InvalidArgumentException